proto/rill/local/v1/api.proto

```diff
@@ -22,6 +22,9 @@ service LocalService {
 
   // Deploy deploys the local project to the Rill cloud.
   rpc Deploy(DeployRequest) returns (DeployResponse) {}
+
+  // UpdateProject updates an existing project.
+  rpc UpdateProject(UpdateProjectRequest) returns (UpdateProjectResponse) {}
```

Comment:
> Could UpdateProject be combined with Deploy?

We already have a very complicated `deploy` command because we are trying to do many things there. I think we should keep the APIs separated. 
I can rename it to `RedeployProject`.

Reply:
Okay sounds good – can you change `Deploy` to `DeployProject` as well then?